README.md

```diff
@@ -556,6 +563,9 @@ Returns the method associated with current auth flow; it might be sms, otp or pu
 ### Enrollment
 
 #### enrollment.getAvailableMethods()
+**DEPRECATED:** Use `.getAvaialbeAuthenticatorTypes` instead. This method does not
+includes recovery authenticator which has become first-class citizen authenticators.
```

Comment:
suggestion: `has become its own specific authenticator type` (or if we want to use the `first-class`, `has become a first-class citizen authenticator`).